plugins/modules/source_control/gitlab/gitlab_service.py

```diff
@@ -0,0 +1,467 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Raphaël Droz <raphael@droz.eu>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: gitlab_service
+short_description: Creates, updates or deletes GitLab services (aka "integrations").
+version_added: '1.3.0'
+description:
+  - Setup or delete GitLab integration services.
```

Comment:
```suggestion
  - Setup or delete GitLab integration services.
```
the short description is longer than the description, you could just make them equal (the final dot must be in a `description` and must not be in a `short_description`)